content/boilerplates/notes/1.2.1.md

```diff
@@ -0,0 +1,16 @@
+## Bug fixes and minor enhancements
+
+- Fix duplicate CRD being generated in the install ([Issue 14976](https://github.com/istio/istio/issues/14976))
+- Allow users to disable istio default retries by setting retries to 0 ([Issue 14900](https://github.com/istio/istio/issues/14900))
```

Comment:
I would be great if we can indicate what the default is (think it is 2 or 3), and how to reset it to 0.

Reply:
cc @howardjohn @GregHanson ^^